0000-charity-welcome-doc.md

```diff
@@ -0,0 +1,34 @@
+# Introducing a new series: Post-Mortem Book Reports
+
+## Dear fellow systems engineers,
```

Comment:
Makes sense, maybe just engineers - either way systems engineers works for me just don't want to exclude anyone.